Styling/README.md

```diff
@@ -540,6 +570,598 @@ ${temperatures.values[0]} === 70
 ${temperatures['values'][0]} === 70 // Same as (temperatures[values])[0] and temperatures.values[0]
 ```
 
+### Built-in Variables
+
+The follow built-in variables are supported by the styling language:
+
+* [`TILES3D_TILESET_TIME`](#TILES3D_TILESET_TIME)
+
+#### TILES3D_TILESET_TIME
```

Comment:
Is this paradigm similar to `czm_` GLSL builtins?  Perhaps it should be `${tiles3d_tileset_time}`